internal/graph/graph.go

```diff
@@ -442,12 +444,33 @@ func ShortenFunctionName(f string) string {
 	f = cppAnonymousPrefixRegExp.ReplaceAllString(f, "")
 	for _, re := range []*regexp.Regexp{goRegExp, javaRegExp, cppRegExp} {
 		if matches := re.FindStringSubmatch(f); len(matches) >= 2 {
-			return strings.Join(matches[1:], "")
+			name := strings.Join(matches[1:], "")
+			if re == goRegExp {
+				return shortenGoFunc(f, name)
+			}
+			return name
 		}
 	}
 	return f
 }
 
+func shortenGoFunc(f string, name string) string {
+	if !goVerRegExp.MatchString(name) {
+		return name
+	}
+
+	// The shortened name could start with a module version (like "v2"). Go back one slash.
+	end := len(f) - len(name) - 1
+	if end >= 0 {
+		prefix := f[:end]
+		if idx := strings.LastIndex(prefix, "/"); idx >= 0 {
+			end = idx
+		}
+	}
+
```

Comment:
Perhaps remove this line break.

Reply:
Sure, if you want. I view the above block as calculating `end`, so that was my thinking.